src/client/pps.go

```diff
@@ -202,28 +207,12 @@ func NewPipeline(pipelineName string) *pps.Pipeline {
 	return &pps.Pipeline{Name: pipelineName}
 }
 
-// CreateJob creates and runs a job in PPS.
-// This function is mostly useful internally, users should generally run work
-// by creating pipelines as well.
-func (c APIClient) CreateJob(pipeline string, outputCommit, statsCommit *pfs.Commit) (*pps.Job, error) {
-	job, err := c.PpsAPIClient.CreateJob(
-		c.Ctx(),
-		&pps.CreateJobRequest{
-			Pipeline:     NewPipeline(pipeline),
-			OutputCommit: outputCommit,
-			StatsCommit:  statsCommit,
-		},
-	)
-	return job, grpcutil.ScrubGRPC(err)
-}
-
 // InspectJob returns info about a specific job.
-// blockState will cause the call to block until the job reaches a terminal state (failure or success).
 // full indicates that the full job info should be returned.
-func (c APIClient) InspectJob(jobID string, blockState bool, full ...bool) (*pps.JobInfo, error) {
+func (c APIClient) InspectJob(pipelineName string, jobID string, full ...bool) (*pps.JobInfo, error) {
 	req := &pps.InspectJobRequest{
-		Job:        NewJob(jobID),
-		BlockState: blockState,
+		Job:        NewJob(pipelineName, jobID),
+		BlockState: false,
```

Comment:
You could just use the default here. Also, I guess there was a decision to expose the block functionality as two separate methods rather than an option on one. What was the thinking behind this decision?

Reply:
Trying to be consistent with out API between PFS and PPS, and also a preference for avoiding boolean parameters if possible because they make code very hard to read.